src/main/java/org/isf/medicalinventory/gui/InventoryBrowser.java

```diff
@@ -592,73 +594,99 @@ public String formatDateTime(GregorianCalendar time) {
 		return format.format(time.getTime());
 	}
 
-	private void ajustWidth() {
-		for (int i = 0; i < pColumwidth.length; i++) {
-			jTableInventory.getColumnModel().getColumn(i).setMinWidth(pColumwidth[i]);
-		}
-	}
-
 	private JComboBox<String> getComboBox() {
 		if (statusComboBox == null) {
 			statusComboBox = new JComboBox<String>();
-			statusComboBox.addItem("");
+			statusComboBox.addItem(MessageBundle.getMessage("angal.common.all.txt"));
 			List<String> statusList = medicalInventoryManager.getStatusList();
 			for (String status : statusList) {
 				statusComboBox.addItem(status);
 			}
 			statusComboBox.addActionListener(actionEvent -> {
-				InventoryBrowsingModel inventoryModel = new InventoryBrowsingModel();
-				totalRows = inventoryModel.getRowCount();
+				totalRows = medicalInventoryManager.getInventoryCount(InventoryType.main.toString());
 				startIndex = 0;
+				int page = startIndex / PAGE_SIZE + 1;
```

Comment:
The result of `startIndex / PAGE_SIZE is always zero (startIndex is 0).   So the result is always 1.